### PR TITLE
common.xml: Add MAV_RESULT_COMMAND_LONG_ONLY and MAV_RESULT_COMMAND_I…

### PR DESCRIPTION
**Pull request by peterbarker**
…NT_ONLY to MAV_RESULT (#1982)

* common.xml: Add MAV_RESULT_COMMAND_LONG_ONLY value to MAV_RESULT enum

* common.xml: Add MAV_RESULT_COMMAND_INT_ONLY value to MAV_RESULT enum

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -2278,6 +2278,12 @@
       <entry value="5" name="MAV_RESULT_IN_PROGRESS">
         <description>Command is valid and is being executed. This will be followed by further progress updates, i.e. the component may send further COMMAND_ACK messages with result MAV_RESULT_IN_PROGRESS (at a rate decided by the implementation), and must terminate by sending a COMMAND_ACK message with final result of the operation. The COMMAND_ACK.progress field can be used to indicate the progress of the operation. There is no need for the sender to retry the command, but if done during execution, the component will return MAV_RESULT_IN_PROGRESS with an updated progress.</description>
       </entry>
+      <entry value="7" name="MAV_RESULT_COMMAND_LONG_ONLY">
+        <description>Command is valid, but it is only accepted when sent as a COMMAND_LONG (as it has float values for params 5 and 6).</description>
```
Comment (Ryanf55) [inline message_definitions/v1.0/common.xml:2282]: Remove `Command is valid` qualifier.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -2278,6 +2278,12 @@
...
+      </entry>
+      <entry value="8" name="MAV_RESULT_COMMAND_INT_ONLY">
+        <description>Command is valid, but it is only accepted when sent as a COMMAND_INT (as it encodes a location in params 5, 6 and 7, and hence requires a reference MAV_FRAME).</description>
+      </entry>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:2286]: When we discussed this with @nexton-winjeel we were focused on the values in location. Fact is, as a consumer of this ACK you might be interested in "why", but it won't affect what you do. So I would be happy with:

- Command is only accepted when sent as a COMMAND_INT
- Command is only accepted when sent as a COMMAND_LONG

Please propose a PR upstream. Doing it here is annoying.

Comment (hamishwillee) [reply message_definitions/v1.0/common.xml:2286]: Ignore that, you already did it!! Merged: https://github.com/mavlink/mavlink/pull/2059

**Comment by nexton-winjeel** [comment]
Thanks @peterbarker!

**Review by tridge — APPROVED** [review]

**Pull request merged**